src/constants/Constants.js

```diff
@@ -1,2 +1,4 @@
 export const SET_PAGE = 'SET_PAGE';
-export const SET_EVENT = 'SET_EVENT';
\ No newline at end of file
+export const SET_EVENT = 'SET_EVENT';
+export const SET_USER = 'SET_USER';
+export const LOGOUT = 'LOGOUT';
```

Comment:
If anyone is curious about the "No newline..." icon: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
do u want me to add a newline

Reply:
nah, was just curious and linked for anyone else who might be wondering